website/content/docs/auth/kubernetes.mdx

```diff
@@ -101,9 +106,91 @@ management tool.
+because Vault stores that value in its own storage and does not automatically
+refresh it.
+
+There are a few different ways to configure Kubernetes auth in this
+scenario, each with their own tradeoffs. This table summarizes the options,
+each of which is explained in more detail below.
+
+| Option                               | All tokens are short-lived | Can revoke tokens early | Other considerations |
+| ------------------------------------ | -------------------------- | ----------------------- | -------------------- |
+| Use client JWT as reviewer JWT       | Yes                        | Yes                     | Operational overhead |
+| Use long-lived token as reviewer JWT | No                         | Yes                     |                      |
+| Use JWT auth                         | Yes                        | No                      |                      |
+
+-> **Note:** By default, Kubernetes currently extends the lifetime of admission
+injected service account tokens to a year to help smooth the transition to
+short-lived tokens. If you would like to disable this, set
+[--service-account-extend-token-expiration=false][k8s-extended-tokens] for
+`kube-apiserver` or specify your own `serviceAccountToken` volume mount at
+`/var/run/secrets/kubernetes.io/serviceaccount` in your pods. See
+[here](/docs/auth/jwt/oidc_providers#specifying-ttl-and-audience) for an example.
+
+[k8s-1.21-changelog]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md#api-change-2
+[k8s-extended-tokens]: https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/#options
+
+#### Use the Vault client's JWT as the reviewer JWT
+
+When configuring Kubernetes auth, you can omit the `token_reviewer_jwt`, and Vault
+will use the Vault client's JWT as its own auth token when communicating with
+the Kubernetes TokenReview API. If Vault is running in Kubernetes, you also need
+to set `disable_local_ca_jwt=true`.
+
+This means Vault does not store any JWTs and allows you to use short-lived tokens
+everywhere but adds some operational overhead to maintain the cluster role
+bindings on the set of service accounts you want to be able to authenticate with
+Vault. Each client of Vault would need the `system:auth-delegator` ClusterRole:
+
+```bash
+kubectl create clusterrolebinding vault-client-auth-delegator \
+  --clusterrole=system:auth-delegator \
+  --group=group1 \
+  --serviceaccount=default:svcaccount1 \
+  ...
+```
+
+#### Continue using long-lived tokens
+
+The default Kubernetes secret created for a service account is still long lived,
+and can be used as the `token_reviewer_jwt` without needing to refresh it. To
+find the secret, run:
+
+```bash
+kubectl get secret "$(kubectl get serviceaccount default -o jsonpath='{.secrets[0].name}')"
+```
+
+Using this maintains previous workflows but does not fully take advantage of the
+new default short-lived tokens.
+
+#### Use JWT auth
```

Comment:
I am having issues with the relevance in this section given that this is for the JWT plugin vs the k8s auth as such it may be better to provider  an "Important note" section as an introductory for this this documentation and redirect users quickly to that method / reading versus potentially discovering this half way down the page.

Reply:
Agreed, I've added a note at the top and updated some of the language to contextualise it better.